packages/@snowpack/app-template-svelte-typescript/tsconfig.json

```diff
@@ -1,11 +1,10 @@
-{
+{ 
+  "extends": "@tsconfig/svelte/tsconfig.json",
   "compilerOptions": {
     "module": "esnext",
     "target": "esnext",
-    "moduleResolution": "node",
     "jsx": "preserve",
     "allowSyntheticDefaultImports": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
I only removed the options that are redundant with `@tsconfig/svelte/tsconfig.json`. @drwpow, you mentioned getting rid of most of the config here. The [blog post](https://svelte.dev/blog/svelte-and-typescript) suggests:
```json
{
  "extends": "@tsconfig/svelte/tsconfig.json",

  "include": ["src/**/*", "src/node_modules"],
  "exclude": ["node_modules/*", "__sapper__/*", "public/*"],
}
```

Those `includes` and `excludes` are probably not what we want though. This might be more appropriate:

```json
{
  "extends": "@tsconfig/svelte/tsconfig.json",
  "compilerOptions": {
    // You can't currently define paths in your 'extends' config,
    // so we have to set 'baseUrl' & 'paths' here.
    // Don't change these unless you know what you're doing.
    // See: https://github.com/microsoft/TypeScript/issues/25430
    "baseUrl": "./",
    "paths": { "*": ["web_modules/.types/*"] },
    "types": ["svelte", "jest", "@testing-library/jest-dom"]
  }
  "include": ["src", "types"],
  "exclude": ["node_modules"]
}
```

What are your thoughts? Any options you'd like to keep from the old config I missed?

Reply:
Nope! That makes sense to me. Thanks for the explanation.

Reply:
Cool, let me know if you would like a follow up with the more minimal config I suggest. Not sure which ones are arbitrary vs. preferred options

Reply:
Ah sorry, I wasn’t being clear. Yes most of the other options are probably best to leave, as they affect Snowpack’s output (by default, TypeScript doesn’t generate very ESM-friendly output, so the other options help with that). I agree with your PR only removing the redundant ones from Svelte’s config—I had assumed there was more to Svelte’s TSConfig than there actually was.

Reply:
No problem! That’s what I was assuming, thanks for clarifying. Glad to get my feet wet with the project 😄